dp/cloud/go/services/dp/storage/db/query_test.go

```diff
@@ -95,7 +95,7 @@ func (s *QueryTestSuite) TestCreateResourceWithAutogeneratedId() {
 			WithBuilder(s.resourceManager.GetBuilder()).
 			From(data).
 			Select(db.NewExcludeMask("id")).
-			Insert()
+			Insert(db.NewExcludeMask("id"))
```

Comment:
Now with this change we should probably remove unused Selects (keep them only for fetch/list/when needed)